ci/cd/run.sh

```diff
@@ -6,7 +6,7 @@ make rebuild-ci-container-image
 # testrunner run tsc which requires buildinfo package to be set
 make set-build-info-constants
 make rebuild-testrunner-container-images
-make rebuild-looker-container-images
+make rebuild-looker-container-image
```

Comment:
Wow, I keep bugging you changes in the upgrade test pipeline. Sorry, I should do more grepping @sreis